src/bio/bio_xstream.c

```diff
@@ -159,6 +159,25 @@ bypass_health_collect()
 	return nvme_glb.bd_bypass_health_collect;
 }
 
+struct bio_faulty_criteria	glb_criteria;
+
+/* TODO: Make it configurable through control plane */
+static inline void
+set_faulty_criteria(void)
+{
+	glb_criteria.fc_enabled = false;
+	glb_criteria.fc_max_io_errs = 5;
+	glb_criteria.fc_max_csum_errs = 5;
+
+	d_getenv_bool("DAOS_NVME_AUTO_FAULTY_ENABLED", &glb_criteria.fc_enabled);
+	d_getenv_int("DAOS_NVME_AUTO_FAULTY_IO", &glb_criteria.fc_max_io_errs);
```

Comment:
Sorry, not understand why need fc_max_io_errs. Based on what consideration when we set it?

Reply:
discussed this with Fanyong offline.